owncloudData/src/test/java/com/owncloud/android/data/storage/ScopedStorageProviderTest.kt

```diff
@@ -0,0 +1,330 @@
+package com.owncloud.android.data.storage
+
+import android.content.Context
+import android.net.Uri
+import com.owncloud.android.domain.transfers.model.OCTransfer
+import com.owncloud.android.testutil.OC_FILE
+import com.owncloud.android.testutil.OC_SPACE_PROJECT_WITH_IMAGE
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import java.io.File
+
+class ScopedStorageProviderTest {
+    private lateinit var scopedStorageProvider: ScopedStorageProvider
+
+    private lateinit var context: Context
+    private lateinit var rootFolderName: String
+    private lateinit var file: File
+    private lateinit var directory: File
+    private lateinit var accountDirectoryPath: String
+    private lateinit var rootFolderPath: String
+    private lateinit var filesDir: File
+
+    private val absolutePath = "/storage/emulated/0/owncloud"
+    private val remotePath = "/storage/emulated/0/owncloud/remotepath"
+    private val spaceId = OC_SPACE_PROJECT_WITH_IMAGE.id
+    private val accountName = "owncloud"
+    private val newName = "owncloudNewName.txt"
+    private val uriEncoded = "/path/to/remote/?x=%D1%88%D0%B5%D0%BB%D0%BB%D1%8B"
+    private val expectedValue: Long = 100
+    private val separator = File.separator
+
+    @Before
+    fun setUp() {
+        context = mockk()
+        filesDir = mockk()
+        rootFolderName = "root_folder"
+        scopedStorageProvider = ScopedStorageProvider(rootFolderName, context)
+        accountDirectoryPath = absolutePath + File.separator + rootFolderName + File.separator + uriEncoded
```

Comment:
Since this variable is just a concatenation of other strings that are defined as properties, this can be a property as well. That way, we have everything ordered 🥳

Reply:
`rootFolderName` and `rootFolderPath` as well 👍

Reply:
Done